.travis.yml

```diff
@@ -13,9 +13,9 @@ stages:
   - name: deploy
     if: type != pull_request AND tag =~ ^v\d+.*
   - name: deployment test setup
-    if: type != pull_request AND tag =~ ^v\d+.*
+    if: (type != pull_request AND tag =~ ^v\d+.*) OR type = cron
```

Comment:
After seeing some issues with the release, I'm thinking it would be useful to have this run nightly as well.